src/sagas/wallet.js

```diff
@@ -596,6 +605,14 @@ export function* onWalletReloadData() {
 
       yield put(tokenInvalidateHistory(tokenUid));
     }
+
+    // We also need to refresh the addresses since we are not starting the wallet again
+    yield call(wallet.getNewAddresses.bind(wallet));
```

Comment:
We don't have this method in the old facade.

Reply:
Fixed in 4250c651855426d27d8ba3e8c155a2462a4a95b2

Thanks